### PR TITLE
Add tests with transaction to empty account, init code containing all valid opcodes.

### PR DESCRIPTION
**Pull request by jwasinger**
Suggested by @winsvega , @cdetrio .  This is still a WIP while I figure out why `testeth` is reporting no errors for all tests in all networks when these tests are filled.

Execution of certain tests (e.g. SUICIDE in transaction init code) should not result in a newly created account as is currently reported when filled.

Files used to generate the filler:
https://gist.github.com/jwasinger/292fd646db4e6759bdb7d25c4fbd22d8

### CODE REVIEW

**Comment by winsvega** [comment]
check with https://github.com/ethereum/tests/blob/develop/src/GeneralStateTestsFiller/stInitCodeTest/TransactionCreateSuicideInInitcodeFiller.json

File: src/GeneralStateTestsFiller/stTransactionTest/Opcodes_TransactionInitFiller.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+{
+    "indexes" : {
+        "data" : -1,
+        "gas" : 0,
+        "value" : 0
+    },
+    "network" : ["ALL"],
+    "result" : {
+        "6295ee1b4f6dd65047762f924ecd367c17eabf8f" : {
+            "shouldexist": 1
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stTransactionTest/Opcodes_TransactionInitFiller.json]: there is no shouldexist. just leave empty account or check the nonce.

**Comment by winsvega** [comment]
when suicide is in init code. no new contract should be created. 
however 
if transaction sending value. suicide(to) address should have those value added

**Comment by jwasinger** [comment]
Still working on this one.  I decided to implement SSTORE in contract init separately so that the case could be addressed: #297

**Comment by jwasinger** [comment]
@pirapira @winsvega Getting closer to getting this one done.  All tests passing except for 'DELEGATECALL' in contract init code.  Hopefully, I can wrap this last case up tomorrow.

Latest version of the script to generate the filler file is at https://github.com/jwasinger/tests/tree/good-opcodes-generation-script/src/GeneralStateTestsFiller/stTransactionTest/tmp.

Feedback on the tests themselves would be appreciated.

**Comment by winsvega** [comment]
@jwasinger  what is the status of this PR?

**Comment by jwasinger** [comment]
Schema issue fixed.  I'm waiting on tests to fill and then the PR will be ready (~10 hours).

**Comment by jwasinger** [comment]
Whoops.  Need to remove BLOCK opcodes from this and refill...

**Comment by jwasinger** [comment]
I've made several mistakes and now I am refilling the tests... again...  It should be ready for merge after filling is finished.  **Note to self**: never put 130 test cases in a single file again....

**Comment by jwasinger** [comment]
@winsvega @pirapira I think this is good now.
